jdk/src/windows/bin/amd64/jvm.cfg

```diff
@@ -34,5 +34,8 @@
 # "-XXaltjvm=<jvm_dir>" option, but that too is unsupported
 # and may not be available in a future release.
 #
--server KNOWN
--client IGNORE
+-j9vm KNOWN
+-hotspot IGNORE
```

Comment:
I suppose this just copied what is done for other platforms, but comparing with the previous values, it seems the ignores should be:
```
-server IGNORE
-client IGNORE
```

Reply:
Should [re]add them to all jvm.cfg files?

Reply:
I think that can be done separately from this commit.